### PR TITLE
es6-destructuring

### PR DESCRIPTION
**Pull request by easton-morris**
image

### CODE REVIEW

File: es6-destructuring/main.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  }
+];
+
+const [book3, book4, book5] = library;
+
+console.log('book3:', book3);
+console.log('book4:', book4);
+console.log('book5:', book5);
+
+const book6 = library[3];
```
Comment (BrettAlbright) [inline es6-destructuring/main.js]: This does get the value, but it is not destructuring. Note the syntax you used above!

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small change!

File: es6-destructuring/main.js (hunk 1 above)
Comment (easton-morris) [reply es6-destructuring/main.js]: image

fixed!

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**